etc/config.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<!--
+/**
+ *
+ * Adyen Payment module (https://www.adyen.com/)
+ *
+ * Copyright (c) 2015 Adyen BV (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <payment>
+            <adyen_paypal_express>
+                <active>0</active>
+                <model>AdyenPaymentPaypalExpressFacade</model>
+                <order_status>pending</order_status>
+                <title>PayPal</title>
+                <allowspecific>0</allowspecific>
+                <sort_order>0</sort_order>
+                <payment_action>order</payment_action>
+                <can_initialize>0</can_initialize>
+                <is_gateway>1</is_gateway>
+                <can_use_checkout>1</can_use_checkout>
+                <can_capture>1</can_capture>
+                <can_capture_partial>1</can_capture_partial>
+                <can_use_internal>0</can_use_internal>
+                <can_refund_partial_per_invoice>1</can_refund_partial_per_invoice>
+                <can_authorize_vault>1</can_authorize_vault>
+                <can_refund>1</can_refund>
+                <can_void>1</can_void>
+                <can_cancel>1</can_cancel>
+                <supports_recurring>1</supports_recurring>
+                <supports_manual_capture>1</supports_manual_capture>
+                <supports_auto_capture>1</supports_auto_capture>
+                <is_wallet>0</is_wallet>
+                <group>adyen-alternative-payment-method</group>
```

Comment:
Shall we assign a different group (`adyen-express-payment-method`) to this payment method to differentiate regular alternative payment methods? This field is being used [here](https://github.com/Adyen/adyen-magento2/blob/da20986ee234d8e6733a3761b45015f16fb0f801/Helper/PaymentMethods.php#L476).